src/capital-project/capital-project.service.ts

```diff
@@ -25,6 +31,46 @@ export class CapitalProjectService {
     return capitalProjects[0];
   }
 
+  async findGeoJsonByManagingCodeCapitalProjectId(
+    params: FindCapitalProjectGeoJsonByManagingCodeCapitalProjectIdPathParams,
+  ) {
+    const capitalProjectCheck =
+      await this.capitalProjectRepository.checkByManagingCodeCapitalProjectId(
+        params.managingCode,
+        params.capitalProjectId,
+      );
+    if (capitalProjectCheck === undefined)
+      throw new ResourceNotFoundException();
```

Comment:
We get the same information from the length check on [line 50](https://github.com/NYCPlanning/ae-zoning-api/pull/354/files#diff-4e2c9606a5354e68c77526366f46e162527158080cadb08b16634a2ec8d9803fR50) that we get here in the dedicated check.

We can save a hit on the database by removing this dedicated check and relying solely on the length check on line 50.  

```suggestion
    const capitalProjectCheck =
      await this.capitalProjectRepository.checkByManagingCodeCapitalProjectId(
        params.managingCode,
        params.capitalProjectId,
      );
    if (capitalProjectCheck === undefined)
      throw new ResourceNotFoundException();
```